docs/cellxgene_census_docsite_schema.md

```diff
@@ -25,17 +25,17 @@ A `SOMAcollection` with tables providing information of the census as a whole, i
 
 - `"summary"`: high-level information of this Census, e.g. build date, total cell count, etc.
 - `"datasets"`: A table with all datasets from CELLxGENE Discover used to create the Census.
-- `"summary_cell_counts"`: Cell counts stratified by relevant cell metadata
+- `"summary_cell_counts"`: Cell counts stratified by relevant cell metadata.
 
 ### Census single-cell data `"census_data"`
 
 Data for each organism is stored in independent `SOMAExperiment` objects which are a specialized form of a `SOMACollection`. Each of these store a data matrix (cell by genes), cell metadata, gene metadata, and feature presence matrix:
 
 This is how the data is organized for one organism – *Homo sapiens*:
 
-- `["homo_sapiens"].obs`: Cell metadata
-- `["homo_sapiens"].ms["RNA"].X`: Data matrices, currently only raw counts exist `X["raw"]`
-- `["homo_sapiens"].ms["RNA"].var`: Gene Metadata
+- `["homo_sapiens"].obs`: Cell metadata.
+- `["homo_sapiens"].ms["RNA"].X`: Data matrices: raw counts in `X["raw"]`, and library-size normalized counts in `X["normalized"]` (only avialble in Census schema V1.1.0 and above).
```

Comment:
Question: will readers of this documentation understand which datasets (census versions/labels) correspond to "schema V1.1.0 and above"?  If not, do we need to add additional information somewhere that provides a mapping between build label and schema version?

Reply:
This is a good point. Currently only the weekly release have V1.1.0. User will have a hard time understanding this. I wonder if there's an automated way to have a page in the doc-site that showcases high level information of all existing data releases at any point in time. I will make a ticket for this.